src/xpcc/architecture/platform/driver/core/cortex/stm32/stm32_dccm.ld.in

```diff
@@ -142,6 +152,7 @@ MEMORY
 	%% endif
 %% endfor
 	RAM (rwx) : ORIGIN = {{ ram_origin[0] }}, LENGTH = {{ ram_sizes | sum }}k
+	BACKUP (rwx) : ORIGIN = 0x40024000, LENGTH = 4k
```

Comment:
Backup SRAM should be moved into the device files like CCM, SRAM{1,2,3}.
At the moment you can get >100% heap availability, since the 4kB of the backup SRAM are added to the available SRAM, but not the total device SRAM.